src/vorta/views/main_window.py

```diff
@@ -106,10 +108,46 @@ def set_progress(self, text=''):
         self.progressText.setText(text)
         self.progressText.repaint()
 
-    def set_log(self, text=''):
+    def set_log(self, text='', ret={}):
         self.logText.setText(text)
         self.logText.repaint()
 
+        msgid = ret.get('msgid')
+        repo_url = ret.get('repo_url')
+        cmd = ret.get('cmd')
+        if cmd is not None and cmd != 'init':
+            # Prevent getting 'None' profile on None case
+            profile = BackupProfileModel.get(name=ret['profile_name']) if ret.get('profile_name') is not None else None
+            if msgid == 'LockTimeout':
+                msg = QMessageBox()
+                msg.setStandardButtons(QMessageBox.Yes | QMessageBox.No)
+                msg.setParent(self, QtCore.Qt.Sheet)
+                msg.setText(
+                    translate(
+                        "MainWindow QMessagebox",
+                        f"The repository at {repo_url} might be in use by another computer. Continue?"))
+                msg.button(QMessageBox.Yes).clicked.connect(lambda: self.break_lock(profile))
+                msg.setWindowTitle(translate("MainWindow QMessagebox", "Repository In Use"))
+                msg.exec_()
+            elif msgid == 'LockFailed':
+                msg = QMessageBox()
+                msg.setParent(self, QtCore.Qt.Sheet)
+                msg.setText(
+                    translate(
+                        "MainWindow QMessagebox",
+                        "You do not have permission to access the repository. Run `sudo chown {} -R {}` from the command line to fix this.")  # noqa
```

Comment:
I used format to try and break up the line length.

Just mentioning a permission error is simple, but it doesn't solve the problem. I'll change it, since the command  breaks in the case of running the command where the backup is on a remote file server (can't change permissions).